data-prepper-plugins/otel-trace-raw-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/oteltrace/OtelTraceRawPrepperConfig.java

```diff
@@ -5,4 +5,6 @@
     static final long DEFAULT_TG_FLUSH_INTERVAL = 180L;
     static final String ROOT_SPAN_FLUSH_DELAY = "root_span_flush_delay";
     static final long DEFAULT_ROOT_SPAN_FLUSH_DELAY = 30L;
+    static final long DEFAULT_TRACE_ID_TTL = 300L;
```

Comment:
nit - Can we get the units included in the variable names? e.g. `_TTL_SECONDS`, `_DELAY_SECONDS`